### PR TITLE
Update a bunch of vendor prices, majority low cost exploitable

### PR DESCRIPTION
**Pull request by isotor**

### CODE REVIEW

**Comment by ShelbyZ** [comment]
Is the issue that these prices are not correct?  Are easy to obtain and should have an overall lower price?

**Comment by TeoTwawki** [comment]
well, we want them retail accurate as much as possible _also_, just in case we're cutting them for exploits sake its worth mentioning.

File: sql/item_basic.sql [hunk 1]
```diff
@@ -1115,8 +1115,8 @@ INSERT INTO `item_basic` VALUES (1227,0,'platinum_nugget','platinum_nugget',12,4
 INSERT INTO `item_basic` VALUES (1228,0,'darksteel_nugget','dst._nugget',12,4,38,0,218);
 INSERT INTO `item_basic` VALUES (1229,0,'adaman_nugget','adaman_nugget',12,4,38,0,333);
 INSERT INTO `item_basic` VALUES (1230,0,'copper_nugget','copper_nugget',12,4,38,0,11);
-INSERT INTO `item_basic` VALUES (1231,0,'brass_nugget','brass_nugget',12,4,39,0,940);
```
Comment (helixhamin) [inline sql/item_basic.sql]: https://ffxiclopedia.fandom.com/wiki/Brass_Nugget
~5gil

File: sql/item_basic.sql [hunk 2]
```diff
@@ -1115,8 +1115,8 @@ INSERT INTO `item_basic` VALUES (1227,0,'platinum_nugget','platinum_nugget',12,4
...
-INSERT INTO `item_basic` VALUES (1232,0,'bronze_nugget','bronze_nugget',12,4,38,0,134);
+INSERT INTO `item_basic` VALUES (1231,0,'brass_nugget','brass_nugget',12,4,39,0,5);
```
Comment (helixhamin) [inline sql/item_basic.sql]: https://ffxiclopedia.fandom.com/wiki/Brass_Nugget
~5gil

**Comment by helixhamin** [comment]
@TeoTwawki All the prices that ffxiclopedia has match the values @isotor is changing

https://ffxiclopedia.fandom.com/wiki/Brass_Nugget - ~5 gil
https://ffxiclopedia.fandom.com/wiki/Bronze_Nugget - ~12 gil
I don't see any resell price on wiki listed for coin cookie.
I don't see any resell price on wiki for scroll of Paralyna
https://www.ffxiah.com/item/4663/aquaveil Stated by someone from levi server - NPCs for 100g
https://ffxiclopedia.fandom.com/wiki/Fire_Threnody - ~102 gil
https://ffxiclopedia.fandom.com/wiki/Ice_Threnody - ~138 gil
https://ffxiclopedia.fandom.com/wiki/Wind_Threnody - ~81 gil
https://ffxiclopedia.fandom.com/wiki/Earth_Threnody - ~44 gil
https://ffxiclopedia.fandom.com/wiki/Lightning_Threnody - ~175 gil
https://ffxiclopedia.fandom.com/wiki/Water_Threnody - ~60 gil
https://ffxiclopedia.fandom.com/wiki/Light_Threnody - ~20 gil
https://ffxiclopedia.fandom.com/wiki/Dark_Threnody - ~31 gil
https://ffxiclopedia.fandom.com/wiki/Triple_Dagger - ~6,871 gil
https://ffxiclopedia.fandom.com/wiki/Pet_Food_Alpha - ~2 gil

**Comment by isotor** [comment]
Checked coin cookie, paralyna and aquaveil live on retail, rest as per the above links. 

@ShelbyZ should be no issue with the price_checker as this only affects the base resale price, nothing to do with vendor sale prices.

**Comment by TeoTwawki** [comment]
> Checked coin cookie, paralyna and aquaveil live on retail, rest as per the above links.
> 
> @ShelbyZ should be no issue with the price_checker as this only affects the base resale price, nothing to do with vendor sale prices.

source citing is good. :+1:  be aware that **occasionally** ffxiclopedia will be way wrong (SE edits things and the site is mostly dead now so they aren't getting updated)

**Pull request merged**